pkg/apis/serving/k8s_validation.go

```diff
@@ -267,7 +277,59 @@ func ValidatePodSpec(ps corev1.PodSpec) *apis.FieldError {
 	return errs
 }
 
+//ValidateMultiContainerPorts validates port when specified multiple containers
```

Comment:
nit: space after `//` here and below in `ValidateSidecarContainer`, `ValidateContainer`.